### ISSUE TITLE
Automatic captcha click 

### ISSUE THREAD

**Issue by emanrdesu** [Adamantcheese/Kuroba#838]
Since I use cookies, the captcha justs let me posts without giving me images to tap.
Basically I'm tapping the captcha for no reason and an update allowing for the removal of this would be nice. If there is a captcha, then so be it, let the images load of whatever and then the user will have to do them, but if there isn't, then it's all good.

**Comment by Adamantcheese** [comment]
Oh believe me, I would've loved it, I tried to implement it, at least so that the pictures would appear immediately if it didn't go through automatically, but Google REALLY doesn't want you messing with captchas.
1) There's a type of captcha that's "invisible" that you can "click" with code. But the site key for that captcha has to allow it. 4chan's doesn't allow it.
2) The captcha box seems to reject all javascript click events on it. One of those "haha I'm not a clickable element but I have FANCY javascript to detect it". Even tried to send a MotionEvent tap to the coordinates of the box, but I couldn't figure out why that didn't work either.

tl;dr: It's the best it's gonna be, unless someone submits a PR that actually does it.

**Comment by emanrdesu** [comment]
Oh I see.
Thank you very much for this app BTW, I might just donate.
